src/utils.h

```diff
@@ -96,7 +96,8 @@ int mft_next_record(struct mft_search_ctx *ctx);
 
 /* MAX_PATH definition was missing in ntfs-3g's headers. */
 #ifndef MAX_PATH
-#define MAX_PATH 1024
+#define MAX_PATH 8192
+//#define MAX_PATH 1024
```

Comment:
Who did you not remove MAX_PATH here ?

Reply:
it's temporary code for test. I'll revert this file.